package-mac.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+mv TEMP-Grabber.app/Contents/MacOS/TEMP-Grabber TEMP-Grabber.app/Contents/MacOS/Grabber
+mkdir -p grabber-release
+ln -s  /Applications grabber-release/Applications 
```

Comment:
What does this symlink do?
I don't see it used afterwards. Is it to automatically install Grabber in `/Applications`?

Reply:
This is related to why we don't directly use macdeployqt -dmg, .dmg files almost always have a symlink to /Applications for easy installation, macdeployqt -dmg does not create the symlink, so it is manually created instead